### ISSUE TITLE
Feature Request: Activate via [Named] Map Note

### ISSUE THREAD

**Issue by GorgoPrimus** [grandseiken/foundryvtt-multilevel-tokens#143]
Right now a zone set to teleport upon map note will teleport anyone if any map note is clicked on within the zone - and that's great. My request though is to set it up so that you can choose to only have it teleport if a map note which a specific name is clicked in the zone. When setting up a teleportation room or some such that can take you to various places, I'd like to be able to make a in zone that covers the entire room and put a map note 'destination 1', and then put another zone right on top of it with a map note for 'destination 2', etc. That way I can easily pick and manage which of the in/outs for that room is being triggered. Right now I flat out can't use a map note because it'd trigger every single one at once if they were on top of each other, and my only real option I've found is to put the different destination Ins in different parts of the room (destination 1 is this corner) which breaks immersion and makes it very hard to manage in a smaller sized room.

tldr: Please add a line under 'Activate via Map Note' where you can choose to enter a name, so that only when clicking on a map note with that same name within the zone will it activate.
